### PR TITLE
Update public-api-server.md

### PR DESCRIPTION
**Pull request by deverka**
Add information about OnFinality

### CODE REVIEW

**Comment by laviniat1996** [comment]
@deverka can you fix linting errors please?

**Comment by deverka** [comment]
Hi @laviniat1996 I updated the code.  Thanks!

File: docs/apis/avalanchego/public-api-server.md [hunk 1]
```diff
@@ -288,3 +288,23 @@ Features:
...
+#### Websockets
+
+- For P-Chain API, the URL is  `https://avalanche.api.onfinality.io/public/ext/bc/P`
+
+Features:
+
+- Powerful API Analytics
+- Full Archive Node Access
+- Globally Distributed Infrastructure
+- Enhanced Performance
```
Comment (yulin-dong) [inline docs/apis/avalanchego/public-api-server.md:309]: This part should not be title case.

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/apis/avalanchego/public-api-server.md (hunk 1 above)
Comment (laviniat1996) [reply docs/apis/avalanchego/public-api-server.md:309]: @deverka can you please check out the above request?

**Comment by meaghanfitzgerald** [comment]
@deverka Please address the comments above or let me know if you are actively working on this PR and need more time. Otherwise, I will close this by EOW. Thanks.

**Comment by laviniat1996** [comment]
Closing this PR due to lack of activity. It has been open for a long time. @deverka If you ever wish to fix the comments, you can re-open the PR.

**Pull request closed without merging**